bika/lims/browser/analysisrequest/publish.py

```diff
@@ -1459,6 +1461,53 @@ def _managers_data(self, ar):
 
         return managers
 
+    @viewcache.memoize
+    def _verifiers_data(self, ar_uid):
+        verifiers = dict()
+        for brain in self.get_analyses(ar_uid):
+            an_verifiers = brain.getVerificators or ''
+            an_verifiers = an_verifiers.split(',')
+            for user_id in an_verifiers:
+                user_data = self._user_contact_data(user_id)
+                if not user_data:
+                    continue
+                verifiers[user_id]=user_data
+        return {'ids': verifiers.keys(),
+                'dict': verifiers}
+
+    @viewcache.memoize
+    def get_analyses(self, ar_uid):
+        query = dict(getRequestUID=ar_uid,
+                     portal_type='Analysis',
+                     cancellation_state='active',
+                     review_state=['verified', 'published'])
+        return api.search(query, CATALOG_ANALYSIS_LISTING)
+
+    @viewcache.memoize
+    def _user_contact_data(self, user_id):
+        user = self.get_user_contact(user_id)
+        if not user:
+            return None
```

Comment:
Just a heads up note:
We miss here verifiers w/o linked user contact. E.g. if someone logged in with a plain user account and verifies the results, this user-id is not listed.